comp/logs/agent/config/endpoints.go

```diff
@@ -68,7 +68,7 @@ func (e *Endpoint) GetStatus(prefix string, useHTTP bool) string {
 
 	var protocol string
 	if useHTTP {
-		if e.UseSSL {
+		if e.UseSSL == nil || *e.UseSSL {
```

Comment:
Maybe we can make a helper function on `Endpoint` that returns `e.UseSSL == nil || *e.UseSSL` so you don't have to duplicate this code in several places

Reply:
Good idea